src/prefect/serialization/environment.py

```diff
@@ -19,9 +19,9 @@ class Meta:
         object_class = LocalEnvironment
 
 
-class CloudEnvironmentSchema(ObjectSchema):
+class DaskKubernetesEnvironmentSchema(ObjectSchema):
```

Comment:
The `min_workers` and `max_workers` need to be serialized as well or else it will just be recreated with the `1` defaults on spin up